src/include/storage/garbage_collector.h

```diff
@@ -75,44 +62,10 @@ class GarbageCollector {
   void UnregisterIndexForGC(common::ManagedPointer<index::Index> index);
 
  private:
-  /**
```

Comment:
I can't leave a comment higher, but why do we still need `PerformGarbageCollection()`? Wouldn't all of that be handled by the actions in the deferred actions framework?

Reply:
Nvm, I see that it just calls deferred action manager `Process` and there's a todo to replace `garbage_collector_thread`